charts/consul/templates/api-gateway-gatewayclassconfig.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if (and .Values.apiGateway.enabled .Values.apiGateway.managedGatewayClass.enabled) }}
+apiVersion: api-gateway.consul.hashicorp.com/v1alpha1
+kind: GatewayClassConfig
+metadata:
+  name: consul-api-gateway-class-config
```

Comment:
```suggestion
  name: consul-api-gateway
```

Reply:
Let me know if this is a breaking change for your team! Ignore it if that is the case.